test/functional/basic/find-e2e-specs.js

```diff
@@ -268,27 +268,27 @@ describe('XCUITestDriver - find', function () {
       await driver.back();
     });
     it('should return all image elements with internally generated ids', async function () {
-      let el = await driver.elementByAccessibilityId('Image View');
+      let el = await driver.$('~Image View');
       await el.click();
 
-      let els = await driver.elementsByClassName('XCUIElementTypeImage');
+      let els = await driver.$$('XCUIElementTypeImage');
       els.length.should.be.above(0);
       for (const el of els) {
-        el.should.exist;
+        el.elementId.should.exist;
       }
     });
 
     describe('textfield case', function () {
       it('should find only one textfield', async function () {
         // TODO: this works locally but fails in CI.
-        if (process.env.CI && UICATALOG_CAPS.platformVersion === '10.3') {
+        if (process.env.CI && UICATALOG_CAPS.alwaysMatch['appium:platformVersion'] === '10.3') {
```

Comment:
prefer  using`extractCapabilityValue` function from desired.js

Reply:
thank you, addressed as https://github.com/appium/appium-xcuitest-driver/pull/1453/commits/33ff8f704eb2e3f53c47fa982262fc9e2015a983